environment.yml

```diff
@@ -1,24 +1,37 @@
 name: geo-ds-cholera
 channels:
+- fastai
 - conda-forge
```

Comment:
Check ordering of channels - Jennifer

Reply:
The channels are checked in the order they are listed, and a dependency is pulled from the first channel in which it is found, unless a version constraint on the dependency cannot be satisified, at which point, the next channel will be checked.  This is the case for the nbdev dependency, which exists in both channels, but since the conda-forge channel doesn't contain the later version specified in the version constraint, it searches the fastai channel, where it finds a suitable version.